modal_proto/api.proto

```diff
@@ -628,11 +628,32 @@ message AuthTokenGetResponse {
   string token = 1;
 }
 
+// Message representing the current (coalesced) state of the autoscaler configuration
+// As well as the different sources that were used to create it.
+message AutoscalerConfiguration {
+  // Used for capturing context about an action performed by a user
+  message UserActionInfo {
```

Comment:
Should this be in global scope? I think the original idea was to have a general message to track "who did what when" that we could reuse elsewhere (although this is the only place we currently use it).